content/collective-communication-pt2.rst

```diff
@@ -150,165 +151,6 @@ Code-along exercise: scatter and gather
    * Download a :download:`working solution <code/collective-communication-scatter-and-gather-solution.c>`
 
 
-All-gather
```

Comment:
Is this moving somewhere else? Or is this going away forever?

Reply:
It's a dupe of content that was incompletely split into pt3